tests/correctness/invoke-memory.expect

```diff
@@ -0,0 +1,16 @@
+{
+  "d": [
+    42,
+    42,
+    42,
+    42,
+    42
+  ],
```

Comment:
One more nit: can you make this data distinct values? Feel free to squash and merge after that.

Reply:
Done. Will merge when tests pass.